src/main/java/com/microexam/classapi/services/ClassService.java

```diff
@@ -0,0 +1,16 @@
+package com.microexam.classapi.services;
+
+import com.microexam.classapi.dto.ClassDTO;
+import com.microexam.classapi.entities.Class;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ClassService {
```

Comment:
Just something to think about or design choice, do we really need a service interface/class?

Reply:
Yes, we do that way we are not exposing the logic of service lauer. We are only exposing the methods and not the implementation